services/web-server/test/helper.js

```diff
@@ -38,6 +39,32 @@ exports.withEntities = (mock, skipping) => {
   withEntity(mock, skipping, exports, 'SessionStorage', SessionStorage);
 };
 
+exports.withPulse = (helper, skipping) => {
+  withPulse({helper, skipping, namespace: 'taskcluster-web-server'});
+};
+
+exports.withMockedEventIterator = () => {
+  let PulseEngineCopy = Object.assign({}, PulseEngine);
+
+  PulseEngineCopy.NextAsyncIterator = null;
+  exports.SetNextAsyncIterator = (asyncIterator) => {
```

Comment:
[nit] this should have a lower-case `s` since it's a function
```suggestion
  exports.setNextAsyncIterator = (asyncIterator) => {
```